### PR TITLE
wrap single digit superscripts and subscripts

### PR DESCRIPTION
**Pull request by builtbywill**

### CODE REVIEW

File: src/addons/outputLatex.js [hunk 1]
```diff
@@ -609,25 +609,12 @@ MathAtom.MathAtom.prototype.toLatex = function(options) {
 
     }
     if (this.superscript) {
-        let sup = latexify(this, this.superscript, options);
-        if (sup.length === 1) {
-            if (sup === '\u2032') {     // PRIME
```
Comment (tragessere) [inline src/addons/outputLatex.js:614]: @hletzrin I think we'll want to keep these prime/double-prime character replacements then add the braces outside the `if` statement

**Comment by tragessere** [comment]
@hletzrin This seems to be working well, but could you revert some of the extra unrelated file changes? The yarn lock file is one we'd like to keep here at least.

**Comment by builtbywill** [comment]
> @hletzrin This seems to be working well, but could you revert some of the extra unrelated file changes? The yarn lock file is one we'd like to keep here at least.

@tragessere @hletzrin I don't think this repo actually uses yarn, so we can delete the yarn.lock file which was added here, and just use `npm install` instead

File: src/addons/outputLatex.js (hunk 1 above)
Comment (builtbywill) [reply src/addons/outputLatex.js:614]: @tragessere good call

**Comment by builtbywill** [comment]
@hletzrin it looks like you might have force pushed, so the updates to dist/ and docs/ are no longer in here. could you run `npm run dist` again and commit those changes?

**Comment by builtbywill** [comment]
@hletzrin i think you may have made your changes in the `dist/src` folder instead of the `src` folder, so I think they got wiped out when you ran `npm run dist`

**Comment by hletzrin** [comment]
> @hletzrin i think you may have made your changes in the `dist/src` folder instead of the `src` folder, so I think they got wiped out when you ran `npm run dist`

You're right, my bad! It should be fixed now

**Review by tragessere — APPROVED** [review]

**Pull request merged**